### DISCUSSION TITLE
How can I enable Feature Gates ?

### DISCUSSION THREAD

**Discussion by jidckii** [kube-hetzner/terraform-hcloud-kube-hetzner#683] [Q&A] [answered]
Hello!
I have a problem getting cadvisor metrics, this is a known issue when migrating to containerd-cri. But in version 1.23 there was a [feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) [PodAndContainerStatsFromCRI](https://kubernetes.io/docs/reference/instrumentation/cri-pod-container-metrics/) , how do I specify it?

**Comment by K-MeLeOn** [comment, marked as answer]
Hi, it seems that you just have to uncomment this [file](https://github.com/kube-hetzner/terraform-hcloud-kube-hetzner/blob/8b37fbed06e488ec218da14861bc3d68ff5d0a72/kube.tf.example#L392) in the kube.tf and add your aguments.
